client/commands/v2/language_server_protocol.py

```diff
@@ -424,3 +424,18 @@ def from_json_rpc_parameters(
         return _parse_parameters(
             parameters, target=WorkspaceDidChangeConfigurationParameters
         )
+
+@dataclasses_json.dataclass_json(
+    letter_case=dataclasses_json.LetterCase.CAMEL,
+    undefined=dataclasses_json.Undefined.EXCLUDE,
+)
+@dataclasses.dataclass(frozen=True)
+class DidCopyModelParameters:
+    path: Path
```

Comment:
This will need to be a str, and then converted to `Path` after, for serialization purposes
```suggestion
    path: str
```